core-services/prow/02_config/_config.yaml

```diff
@@ -711,7 +711,8 @@ tide:
     - 'dco-signoff: no'
     orgs:
     - cri-o
-  - includedBranches:
```

Comment:
@navidshaikh should this file be changed as well?

Reply:
:+1: 

fixed in the latest commit

Reply:
earlier that file wasnt in the pr but then the CI job would complain, ran `make prow-config` and added that file